go/billing/models.py

```diff
@@ -147,6 +149,11 @@ def calculate_credit_cost(cls, message_cost, storage_cost, markup_percent,
         base_cost = message_cost + storage_cost
         if session_created:
             base_cost += session_cost
+        if session_length and session_unit_length and session_unit_cost:
+            units = (
+                session_length / session_unit_length
+                ).to_integral_exact(rounding=ROUND_CEILING)
```

Comment:
Based on the slack #vumi-mgmt discussion, we should ceil the value here, I think.